src/CFamily.UnitTests/Subprocess/ProtocolTest.cs

```diff
@@ -67,6 +67,22 @@ public void Read_Response()
             response.Messages[0].Parts.Length.Should().Be(1);
         }
 
+        [TestMethod]
+        public void Response_WithDataFlow_DataFlowIgnored()
```

Comment:
I wonder if it would actually be easier to have a DataFlows property on the message, and to populate it? This test is a bit weak as the assertions don't test anything related to data flows. Perhaps it would be easier to populate the flows and assert them.

Reply:
It would increase the scope of the testing and the code with no real benefit it would be discarded anyway. The purpose of this test is to just make sure that we correctly discard the data flows.

Reply:
My problem with the test is that you can change `MockResponseWithDataFlow` to `MockResponseWithQuickFixes` and nothing would fail. There is no assertion anywhere that the data flows were actually attempted to be parsed. I'm not sure how we can assert it otherwise, without actually populating the fields. Do you have any suggestions? :/

Reply:
This would require we would change the Message class and its ctor, which is referenced in 35 places (albeit almost all of them are tests). I do not see the need to assert that data is correctly parsed, as we do not care about it. We just need to assert that the existence of this data will not break our process, and we are asserting that.